spec/support/login_helper.rb

```diff
@@ -0,0 +1,14 @@
+module LoginHelper
+  def login_user(user)
+    visit new_user_session_path
+
+    fill_in "Email", with: user.email
+    fill_in "Password", with: user.password
+
+    click_button "Log in"
+  end
```

Comment:
Clean DRYing of login code :+1: 

Currently `login_user(user)` won't complain if the user fails login (e.g. wrong password). What expectation could be added to `login_user(user)` to check the user has been successfully logged in?